dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/TCPUtils.java

```diff
@@ -0,0 +1,312 @@
+                    framework.getActiveChannelsMap().put(channel, new DefaultChannelGroup(GlobalEventExecutor.INSTANCE));
+                }else {
+                	framework.getOutboundConnections().add(channel);
+                }
+
+                // set common channel attrs
+                channel.attr(ConfigConstants.NAME_KEY).set(config.getExternalName());
+                channel.attr(ConfigConstants.HOST_KEY).set(newHost);
+                channel.attr(ConfigConstants.PORT_KEY).set(inetPort);
+                channel.attr(ConfigConstants.IS_INBOUND_KEY).set(config.isInbound());
+                
+                // Listener to stop channel on close
+                // This should just log that the channel stopped
+                channel.closeFuture().addListener(innerFuture -> logChannelStopped(channel));
+
+                // set up a helpful log message
+                String hostLogString = newHost;
+                SocketAddress addr = channel.localAddress();
+                InetSocketAddress inetAddr = (InetSocketAddress)addr;
+                String IPvType = "IPv4";
+                if (inetAddr.getAddress() instanceof Inet6Address) {
+                    IPvType = "IPv6";
+                }
+                if (newHost == NettyConstants.INADDR_ANY) {
+                    hostLogString = "*  (" + IPvType + ")";
+                } else {
+                    hostLogString = config.getHostname() + "  (" + IPvType + ": "
+                               + inetAddr.getAddress().getHostAddress() + ")";
+                }
+
+                if(TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                	Tr.debug(tc, "serverSocket getInetAddress is: " + inetAddr);
+                    Tr.debug(tc, "serverSocket getLocalSocketAddress is: " + channel.localAddress());
+                    Tr.debug(tc, "serverSocket getInetAddress hostname is: " + inetAddr.getAddress().getHostName());
+                    Tr.debug(tc, "serverSocket getInetAddress address is: " + inetAddr.getAddress().getHostAddress());
+                    Tr.debug(tc, "channelConfig.getHostname() is: " + config.getHostname());
+                    Tr.debug(tc, "channelConfig.getPort() is: " + config.getPort());
+                }
+
+                if (config.isInbound()) {
+                    Tr.info(tc, TCPMessageConstants.TCP_CHANNEL_STARTED,
+                            new Object[] { config.getExternalName(), hostLogString, String.valueOf(inetPort) });
+                } else {
+                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                        Tr.debug(tc, TCPMessageConstants.TCP_CHANNEL_STARTED,
+                                new Object[] { config.getExternalName(), hostLogString, String.valueOf(inetPort) });
+                    }
+                }
+            } else {
+                if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                    Tr.debug(tc,
```

Comment:
If an inbound success is logged as a info I think a failure is a least an info/warning rather than debug?

Reply:
The error is still logged as expected a few line after this one if no retries are left. As for the using info/debug and error/debug I believe is to follow the same logging type as CHFW

Reply:
OK